apps/passport-server/src/services/generic-issuance/pipelines/LemonadePipeline.ts

```diff
@@ -834,7 +834,9 @@ export class LemonadePipeline implements BasePipeline {
 
       const ticketActions: PCDAction[] = [];
 
-      if (this.loaded) {
+      // If the Atom DB is not empty, i.e. if we have loaded atoms for this pipeline
+      // at least once since the server started, delete the folder.
+      if ((await this.db.load(this.id)).length > 0) {
```

Comment:
my suboptimal implementation merged from https://github.com/proofcarryingdata/zupass/pull/1836 - i think i have a solution that is passable and will put it in soon, unless someone beats me to it!

Reply:
I've updated the implementation [here](https://github.com/proofcarryingdata/zupass/pull/1839/commits/95ce7126651827062017134624f9f299a837ff23)